release-notes.html.md.erb

```diff
@@ -1,709 +1,38 @@
-of `bosh-system-metrics-forwarder` will need to be updated to use `system-metrics-agent` as the `origin` label value.
+* **[Breaking Change]** Changes might be observed in Healthwatch metrics on switching to OTel collector for metric collection. All such changes have been described in
+this section. Metrics that were coming from Firehose-specific components like reverse log proxy will not be available after enabling OTel collector. Currently the OTel
+collector is collocated on TAS VMs and will only collect metrics from these VMs. As a result of this, metrics from other tiles will not be collected by the OTel collector. The
+prometheus exporter in OTel collector does not allow leading underscore, except in private metrics. So the OTel collector is removing any leading underscores in metric
+names. Also SVM forwarder VM will not work with otel and will have to be switched off if it is switched on.
 
-* **[Feature]** Added support of VMware Tanzu Kubernetes Grid Integrated Edition v1.16.
+* **[Feature]** Healthwatch 2.3.1 has support for using OTel collectors (instead of Firehose) for collecting metrics. Healthwatch automatically switches to collecting metrics
+from OTel collector when the option on TAS for Enabling Firehose v1/v2 is disabled.
 
-Healthwatch v2.2.9 uses the following open-source component versions:
-
-| Component    | Packaged Version |
-|--------------|------------------|
-| Prometheus   | 2.46.0           |
-| Grafana      | 9.5.9            |
-| Alertmanager | 0.26.0           |
-| PXC          | 1.0.14           |
-
-### <a id='2-2-8'></a> v2.2.8
-
-**Release Date:** August 17, 2023
-
-* **[Feature]** Added a `Time Intervals` config in the Healthwatch tile in `Alertmanager` pane that specifies
-a named interval of time that may be referenced in the routing tree to mute/activate particular routes for particular times of the day.
-For more information, see [Prometheus Documentation](https://prometheus.io/docs/alerting/latest/configuration/#time_interval:~:text=active_time_intervals%3A%0A%20%20%20%20%20%20%2D%20offhours%0A%20%20%20%20%20%20%2D%20holidays-,%3Ctime_interval%3E,-A%20time_interval%20specifies).
-
-* **[Bug Fix]** Fixed `user already exists` error occurs during UAA authentication after upgrading Healthwatch to a newer version.
-For more information, see [User already exists error during UAA authentication after upgrading Healthwatch](#user-already-exists-error) below.
-
-Healthwatch v2.2.8 uses the following open-source component versions:
-
-| Component    | Packaged Version |
-|--------------|------------------|
-| Prometheus   | 2.46.0           |
-| Grafana      | 9.5.7            |
-| Alertmanager | 0.25.0           |
-| PXC          | 1.0.14           |
-
-### <a id='2-2-7'></a> v2.2.7
-
-**Release Date:** July 17, 2023
+* **[Feature]** Healthwatch 2.3.1 has been upgraded to Grafana 10, hence providing the new features offered by Grafana 10.
```

Comment:
Minor suggestion:
* **[Feature]** Healthwatch 2.3.1 is upgraded to Grafana 10 and provides the new features offered by Grafana 10.